src/generic/canbus.c

```diff
@@ -90,8 +91,13 @@ console_sendf(const struct command_encoder *ce, va_list args)
 // Available commands and responses
 #define CANBUS_CMD_QUERY_UNASSIGNED 0x00
 #define CANBUS_CMD_SET_NODEID 0x01
+#define CANBUS_CMD_REQUEST_BOOTLOADER 0x02
 #define CANBUS_RESP_NEED_NODEID 0x20
 
+// CanBoot definitions
+#define CANBOOT_SIGNATURE 0x21746f6f426e6143
+#define CANBOOT_REQUEST   0x5984E3FA6CA1589B
```

Comment:
Just out of curiosity, what's the background to 0x5984E3FA6CA1589B?  The 0x21746f6f426e6143 is "CanBoot!" in ascii - which will show up nicely in a hexdump.

Reply:
It was random.  I can create an ascii signature if you prefer.